tools/wltests/test_series

```diff
@@ -690,64 +825,28 @@ flash_sha1() {
 	build_sha1 "$COMMIT_SHA1" "$COMMIT_DESC"; RESULT=$?
 	[ $RESULT -eq $OK ] || return $RESULT
 
-	### Reboot device into BOOTLOADER
-	echo
-	c_info "Rebooting device into bootloader..."
-	attempts=3
-	[ $DRYRUN -eq 1 ] || reboot_fastboot
-	[ $DRYRUN -eq 1 ] || \
-	while [[ $? -eq $ERROR ]]; do
-		let attempts--
-		if [[ $attempts -eq 0 ]]; then
-			c_error "device not entering FASTBOOT mode"
-			exit $EIO
-		fi
-		c_warning "Failed entering FASTBOOT mode, $attempts remaining attempts..."
-		reboot_fastboot
-	done
-
 	### Flash generated IMAGES
+	[ $DRYRUN -eq 0 ] || return $OK
```

Comment:
We have a lot of tests on `$DRYRUN` after so do we need this one ?

Reply:
I was considering a cleanup patch... but it's out of scope for this PR.
We can certainly add one as a follow up.